chia/consensus/blockchain.py

```diff
@@ -194,6 +194,24 @@ def get_peak(self) -> Optional[BlockRecord]:
             return None
         return self.height_to_block_record(self._peak_height)
 
+    def get_tx_peak(self) -> Optional[BlockRecord]:
+        """
+        Return the most recent transaction block. i.e. closest to the peak of the blockchain
+        Requires the blockchain to be initialized and there to be a peak set
+        """
+
+        if self._peak_height is None:
+            return None
+        tx_height = self._peak_height
+        tx_peak = self.height_to_block_record(tx_height)
+        while not tx_peak.is_transaction_block:
+            if tx_height == 0:
+                return None
```

Comment:
would it make sense to add a `pragma: no cover` for this? as far as I can tell, the chains we generate with block tools all start with a transaction block

Reply:
With a comment to document this assumption, I think that sounds reasonable.

Reply:
Closing because the conversation blocks the merge button and it looks like this was resolved anyway..